src/main/react-webapp/src/components/DrugPage.js

```diff
@@ -24,8 +24,13 @@ const DrugPage = () => {
         <p><b>Classification:</b> {drugInfo["classification"]}</p>
         <DrugPageDesc title="Description" desc={drugInfo["description"]}/>
         <DrugPageList title="Aliases" listItems={drugInfo["aliases"]}/>
+        <DrugPageDesc title="Safety Description" desc={drugInfo["safetyDescription"]}/>
+        <DrugPageList title="Drug Warning Signs" listItems={drugInfo["drugWarningSigns"]}/>
+        <DrugPageDesc title="Description" desc={drugInfo["description"]}/>
```

Comment:
This looks like a duplicate of description above.

Reply:
should be updated now